.gitignore

```diff
@@ -34,3 +34,4 @@ build/
 
 .idea/*
 target/*
+/src/main/resources/reviews.csv
```

Comment:
I can see .idea/ and target/ already indicated above
let's maybe add `logs` folder?

Reply:
done